### PR TITLE
Obelisk the skycoin consensus algorithm.pt.md

### PR DESCRIPTION
**Pull request by SerggioC**

### CODE REVIEW

File: content/statement/Obelisk The Skycoin Consensus Algorithm.pt.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+
+## Prova de trabalho e sistema da Bitcoin
+
+Foi um erro de cálculo fundamental na programação inicial da Bitcoin que
+o processo de mineração produziria uma estrutura de incentivo económico que
+promoveria a descentralização. Na verdade, a ligação entre consenso e hash 
+power incentiva a compra de capacidade de processamento cada vez maior para
+controlar a rede de consenso.
+
```
Comment (marcosbwr) [inline content/statement/Obelisk The Skycoin Consensus Algorithm.pt.md]: económico -> econômico (pt_br)

File: content/statement/Obelisk The Skycoin Consensus Algorithm.pt.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+controlo da rede já foi identificado por Satoshi como a principal ameaça 
+não criptográfica à estabilidade da rede. Permite a atores acumular poder 
+de processamento suficiente e atingir uma taxa de hashing maioritária para
+falsificar ou reverter transações na rede num ataque de 51%. Alguns argumentam que
+esta vulnerabilidade tornou-se menos urgente num ambiente onde o poder de hash
+é altamente centralizado com atores que investiram grandes somas na rede Bitcoin e 
+dependem do alto valor da moeda para a sua sobrevivência. Ainda assim o poder de 
+influenciar a rede está altamente concentrado, derrotando o propósito de uma 
+cryptomoeda baseada num livro-razão.
+
```
Comment (marcosbwr) [inline content/statement/Obelisk The Skycoin Consensus Algorithm.pt.md]: controlo -> controle (pt-br)

File: content/statement/Obelisk The Skycoin Consensus Algorithm.pt.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+na rede determina a sua autoridade e poder de voto para implementar mudanças técnicas
+na mesma. Os participantes podem minerar uma parcela equivalente à sua participação, 
+independentemente do poder de processamento.
+
+Este princípio aumenta significativamente as barreiras económicas para lançar 
+um ataque de 51% porque o custo financeiro de adquirir a maioria dos tokens 
+na rede no mercado aberto é muito provável que exceda o ganho potencial. 
+Se um atacante terminar como o participante maioritário da rede, ele sofrerá 
+mais do impacto do ataque na estabilidade da rede ou do valor externo da criptomoeda.
+
```
Comment (marcosbwr) [inline content/statement/Obelisk The Skycoin Consensus Algorithm.pt.md]: "majority" traduz-se "majoritário" (pt_br).

**Comment by gz-c** [comment]
is this ready? need sky address

**Comment by SerggioC** [comment]
Yes, latest commit ready to publish.
dbovLCsAemG89io81y82CW5CvjFrnocxkF

**Comment by SerggioC** [comment]
@gz-c

**Pull request merged**